tests/test_il_elections.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import BOARD, PASSED, TENTATIVE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+from scrapy.http import HtmlResponse
+
+from city_scrapers.spiders.il_elections import IlElectionsSpider
+
+freezer = freeze_time("2019-12-04")
+freezer.start()
+
+test_response = file_response(
+    join(dirname(__file__), "files", "il_elections_minutes.html"),
+    url="https://www.elections.il.gov/AboutTheBoard/MeetingMinutesAll.aspx",
+)
+
+with open(join(dirname(__file__), "files", "il_elections_agenda.html"), "r") as f:
```

Comment:
You can copy the `test_response` code above and apply it here since it does a lot of what the next lines do for you already